collection_requirements.rst

```diff
@@ -104,20 +104,23 @@ Versioning and deprecation
 * We recommend to announce your policy of releasing, versioning and deprecation to contributors and users in some way. For an example of how to do this, see `the announcement in community.general <https://github.com/ansible-collections/community.general/issues/582>`_. You could also do this in the README.
 
 
-Repository management
-=====================
-
-Repository name
----------------
+Naming
+======
 
 For collections under ansible-collections the repository SHOULD be named ``NAMESPACE.COLLECTION``.
 
 For collections created for working with a particular entity, they should contain the entity name, for example ``community.mysql``.
 
-We should avoid:
+For corporate maintained collections, the repository can be named ``COMPANY_NAME.PRODUCT_NAME``, for example ``ibm.db2``.
+
```

Comment:
```suggestion
`Namspace limitations <https://galaxy.ansible.com/docs/contributing/namespaces.html#galaxy-namespaces>`_  lists requirements for namespaces in Galaxy.
```

Reply:
@samccann commited, thanks!

Reply:
@samccann i changed the reference a little bit to point to the limitations directly